core/domain/feedback_domain.py

```diff
@@ -37,16 +37,32 @@ class FeedbackThread(python_utils.OBJECT):
         summary: str. A summary of the feedback thread.
         has_suggestion: bool. Whether the feedback thread includes a
             suggestion.
+        message_count: int. The number of messages posted onto the thread.
         created_on: datetime.datetime. The time when the feedback thread was
             created.
         last_updated: datetime.datetime. The time when the feedback thread
             was last updated.
+        updated_status: str. The change in status since the last message, if
+            any.
+        last_message_id: int. The id of the last message in the thread.
+        last_message_text: str. The text of the last message in the thread.
+        last_message_author_id: str. The author id of the last message in the
+            thread.
+        second_last_message_id: int. The id of the second-to-last message in the
```

Comment:
Surely this can be derived from the last_message_id? In fact, I would have thought that both of these message_ids can be derived from the message_count, so you don't need to store them...

Reply:
Done.